mindee/geometry/__init__.py

```diff
@@ -0,0 +1,7 @@
+from .bbox import *
+from .error import *
+from .minmax import *
+from .point import *
+from .polygon import *
+from .polygon_utils import *
+from .quadrilateral import *
```

Comment:
No star imports
No relative imports

> Note that in general the practice of importing * from a module or package is frowned upon, since it often causes poorly readable code.

https://docs.python.org/3/tutorial/modules.html

> The python-dev community chose absolute imports as the default because they’re the more common use case and because absolute imports can provide all the functionality of relative (intra-package) imports

https://peps.python.org/pep-0328/